radio/src/targets/common/arm/stm32/usb_driver.cpp

```diff
@@ -79,6 +79,11 @@ void usbStart()
 #endif
 #if defined(USB_SERIAL)
     case USB_SERIAL_MODE:
+//OW
+#if defined(TELEMETRY_MAVLINK)
+    case USB_MAVLINK_MODE:
```

Comment:
This is most probably not necessary, but I might be wrong.

Reply:
it won't hurt, so it's not necessary

Reply:
done